geotrek/tourism/locale/en/LC_MESSAGES/django.po

```diff
@@ -18,240 +18,368 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+#: geotrek/tourism/apps.py:8
+#: geotrek/tourism/templates/tourism/tourism_detail_fragment.html:6
+#: geotrek/tourism/templates/tourism/tourism_extrabody_fragment.html:13
 msgid "Tourism"
 msgstr ""
 
+#: geotrek/tourism/filters.py:28 geotrek/tourism/filters.py:29
+#: geotrek/tourism/filters.py:70 geotrek/tourism/filters.py:71
+#: geotrek/tourism/models.py:71 geotrek/tourism/models.py:276
+#: geotrek/tourism/models.py:450
+#: geotrek/tourism/templates/tourism/touristiccontent_detail_attributes.html:86
+#: geotrek/tourism/templates/tourism/touristicevent_detail_attributes.html:134
```

Comment:
don't add translation location in files. See makefile -> messages command.
All translation files should be updated